Entities/Actor.h

```diff
@@ -1303,6 +1303,30 @@ ENTITYALLOCATION(Actor)
 	void SetTravelImpulseDamage(float value) { m_TravelImpulseDamage = value; }
 
 
+	/// <summary>
+	/// Returns the defined sound to be played on death.
+	/// </summary>
+	/// <returns>A sound with the death sample of this actor.</returns>
+	Sound GetDeathSound() const { return m_DeathSound; }
+
+
+	/// <summary>
+	/// Sets new death sound sample from specified path or sets null.
+	/// </summary>
+	/// <param name="samplePath">Filepath to new sample or None/nil for no sound.</param>
+	/// <returns>Updated DeathSound.</returns>
+	Sound SetDeathSound(char *samplePath) {
+		Sound snd;
```

Comment:
I'd rather name this deathSound than snd, but that's cause I don't much care for the "we refuse to even spell right cause we have to save all the characters" style of naming. Not ultimately important though.
What is more important is where is the "nil" coming from? The lua binding doesn't spit out nils as strings does it? Have you tested it fine with lua?